go/vt/sqlparser/constants.go

```diff
@@ -478,6 +478,9 @@ const (
 	// KillType strings
 	ConnectionStr = "connection"
 	QueryStr      = "query"
+
+	// GroupConcatDefaultSeparator is the default separator for GroupConcatExpr.
+	GroupConcatDefaultSeparator = ","
 )
```

Comment:
nit: it is not used in this package. probably belongs to planbuilder.

Reply:
No, i don't think so. It feels like a sqlparser property even if not used in the package.